src/index.ts

```diff
@@ -1,9 +1,12 @@
 export type ComboboxSettings = {
   tabInsertsSuggestions?: boolean
-  defaultFirstOption?: boolean
+  firstOptionSelectionMode?: FirstOptionSelectionMode
   scrollIntoViewOptions?: boolean | ScrollIntoViewOptions
 }
 
+// Indicates the default behaviour for the first option when the list is shown.
+export type FirstOptionSelectionMode = 'none' | 'selected' | 'focused'
```

Comment:
I think `focused` is a little misleading here because focus actually remains inside the input the whole time.


Even though I don't think the [accessibility docs'](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/command-palette/list-semantics.md#navigating-items) active/selected terminology is super intuitive, it's probably the best we've got - maybe we should stick with that for consistency? This is also consistent with the concept of 'selected' elsewhere in this file meaning the active descendant. That would mean:

- replace `selected` with `active`
- replace `focused` with `selected`

Maybe we can also add a comment with a link to the docs to help understand the terms, since these aren't universal concepts.

Reply:
This is great feedback, I'll update this, thanks!

Reply:
Addressed in: https://github.com/github/combobox-nav/pull/81/commits/3995d9d201e2178b907255cca2edb1f67a771ddd ✋